tests/unit/models/category-test.js

```diff
@@ -1,10 +1,14 @@
 import { moduleForModel, test } from 'ember-qunit';
 import { testForHasMany } from '../../helpers/relationship';
 import '../../helpers/has-attributes';
+import Ember from 'ember';
 
 moduleForModel('category', 'Unit | Model | category', {
   // Specify the other units that are required for this test.
-  needs: ['model:user-category', 'model:project-category']
+  needs: [
+    'model:project-category',
+    'model:user-category',
```

Comment:
As more `eslint` rules get added, this trailing comma may be caught since it is unnecessary. Thoughts on deleting?